assets/package-lock.json

```diff
@@ -1057,13 +1057,13 @@
     "@types/q": {
       "version": "1.5.2",
       "resolved": "https://registry.npmjs.org/@types/q/-/q-1.5.2.tgz",
-      "integrity": "sha512-ce5d3q03Ex0sy4R14722Rmt6MT07Ua+k4FwDfdcToYJcMKNtRVQvJ6JCAPdAmAnbRb6CsX6aYb9m96NGod9uTw==",
+      "integrity": "sha1-aQoUdbhPKohP0HzXl8APXzE1bqg=",
```

Comment:
@joaquinalcerro Yeah, you should probably reset this file. Sounds like you may have a dated version of npm installed as it switched the checksums from sha512 to sha1 which is definitely a downgrade 😁